### ISSUE TITLE
nuget package wexflow version .net Core

### ISSUE THREAD

**Issue by abergeronCofomo** [aelassas/Wexflow#81]

**Comment by abergeronCofomo** [comment]
Is there a nuget package of wexflow that handles .net Core?  I tried to install the wexflow library Wexflow 3.8 to my .net Core library project  and I got the following warning:

Warning	NU1701	Package 'Wexflow 3.8.0' was restored using '.NETFramework,Version=v4.6.1' instead of the project target framework '.NETCoreApp,Version=v2.1'. This package may not be fully compatible with your project.	TacheAPPUI	C:\dev\.net WexFlow\POC_Wexflow\TacheAPPUI\TacheAPPUI\TacheAPPUI.csproj		

Audrey 
Thank you
